go/host/p2p/p2p.go

```diff
@@ -9,33 +9,31 @@ import (
 	"sync/atomic"
 	"time"
 
+	"github.com/ethereum/go-ethereum/rlp"
+	"github.com/obscuronet/go-obscuro/go/common"
 	"github.com/obscuronet/go-obscuro/go/common/host"
-
-	gethlog "github.com/ethereum/go-ethereum/log"
-
 	"github.com/obscuronet/go-obscuro/go/common/log"
-
 	"github.com/obscuronet/go-obscuro/go/config"
 
-	"github.com/ethereum/go-ethereum/rlp"
-	"github.com/obscuronet/go-obscuro/go/common"
+	gethlog "github.com/ethereum/go-ethereum/log"
 )
 
 const (
 	tcp = "tcp"
-)
 
-// A P2P message's type.
-type msgType uint8
-
-const (
 	msgTypeTx msgType = iota
 	msgTypeBatches
 	msgTypeBatchRequest
 )
 
+var _peerListCoolOffPeriod = 10 * time.Second
```

Comment:
Underscore prefix convention isn't widely used in our codebase, but I guess it probably should be. Will start changing to this when I see it. You use it for package-private constants?

Reply:
Yes, exactly. Think it makes clear that it's a package-private global variable/const.

From here https://github.com/uber-go/guide/blob/master/style.md#prefix-unexported-globals-with-_ 

Rationale: Top-level variables and constants have a package scope. Using a generic name makes it easy to accidentally use the wrong value in a different file.